infra/storage/kFile.class.php

```diff
@@ -563,14 +563,28 @@ public static function checkIsDir($path)
 		return is_dir($path);
 	}
 
-	public static function getLineFromFileTail($file , $lineNum)
+	public static function getLineFromFileTail($file)
```

Comment:
rename - getLinesFromFileTail so it can be more generic
add param numberOfLines (default 1)
add param ignoreEmptyEnds = true //this flag indicate if we need ignore ending with new lines.